src/components/AppProvider/utilities/withAppProvider/tests/withAppProvider.test.tsx

```
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import withAppProvider from '../withAppProvider';
+
+describe('withAppProvider', () => {
+  it('throws when polaris is not defined', () => {
+    const consoleSpy = jest
```

this is to suppress the error message being throw